.github/workflows/shared-terraform-chatops.yml

```
@@ -704,7 +685,14 @@ jobs:
           name: state
           path: ${{ needs.context.outputs.terraform_state_path }}
 
-
+      - uses: actions/create-github-app-token@v1
+        if: ${{ needs.context.outputs.uses_github == 'true' }}
```

actionlint (1.7.4)

property "uses_github" is not defined in object type {atmos_version: string; base: string; base_repo_name: string; base_repo_owner: string; body: string; closed_at: string; created_at: string; found: string; go_mod_path: string; go_sum_path: string; head_repo_name: string; head_repo_owner: string; head_sha: string; json: string; labels: string; merged_at: string; number: string; terraform_state_path: string; terraform_state_retention_days: string; test_aws_role: string; test_needs_aws: string; test_needs_fixtures: string; test_skip_concurrency: string; title: string; url: string}

(expression)